### DISCUSSION TITLE
get RestrictionsManager from Android

### DISCUSSION THREAD

**Discussion by trkas** [appcelerator/titanium_mobile#13237] [Q&A]
Hi,

How can I access the RestrictionsManager object in Android from Titanium?

It's for getting current managed configurations on app.

Best regards 
Troels

**Comment by m1ga** [comment]
think you could try to use that with hyperloop

**Reply by trkas** [reply]
OK I don't know so much about Hyperloop, but doesn't it require dependency of the Cloud services that will be discontinued soon?

**Reply by m1ga** [reply]
no, you can still use it. Have a look at examples here in the repo: https://github.com/appcelerator/hyperloop-examples/blob/master/app/controllers/android/touches.js and the documentation at https://developer.android.com/reference/android/content/RestrictionsManager or some examples from Google.

**Reply by trkas** [reply]
OK thank you. I will look further into it :)

**Comment by m1ga** [comment]
To update this thread with the hyperloop code:

```js
const RestrictionsManager = require('android.content.RestrictionsManager');
const Context = require('android.content.Context');
const Activity = require('android.app.Activity');

const activity = new Activity($.index.activity);
var myRestrictionsMgr = RestrictionsManager.cast(activity.getSystemService(Context.RESTRICTIONS_SERVICE));
var appRestrictions = myRestrictionsMgr.getApplicationRestrictions();
console.log(appRestrictions.containsKey("keyName");
```
